### PR TITLE
Update redis kafka docs

### PR DESCRIPTION
**Pull request by moarychan**
Update Redis & Kafka docs
- https://github.com/Azure/azure-sdk-for-java/issues/26550

### CODE REVIEW

**Comment by backwind1233** [comment]
please do not remove cache/.gitignore

File: cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/terraform/variables.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  default     = "eastus"
+}
+
+variable "sample_tag_value" {
+  type        = string
+  description = "The value of spring-cloud-azure-sample tag."
+  default     = "true"
+}
+
+variable "azure_subscription_id" {
```
Comment (backwind1233) [inline cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/terraform/variables.tf:19]: I don't think it's a good approach to do this.

Comment (backwind1233) [reply cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/terraform/variables.tf:19]: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/subscriptions

File: cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/terraform/main.tf [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    terraform                 = "true"
+    application-name          = var.application_name
+    spring-cloud-azure-sample = var.sample_tag_value
+  }
+}
+
+# =================== redis ================
+resource "azurecaf_name" "azurecaf_name_dns_redis" {
+  name          = var.application_name
+  resource_type = "azurerm_eventhub_namespace"
```
Comment (backwind1233) [inline cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/terraform/main.tf:39]: update resource_type here

**Comment by moarychan** [comment]
Closing this in favor of https://github.com/Azure-Samples/azure-spring-boot-samples/pull/165

**Comment by moarychan** [comment]
Beta 3 is also necessary.

**Review by backwind1233 — APPROVED** [review]
LGTM

**Pull request merged**